src/index.ts

```diff
@@ -30,12 +32,11 @@ app.use(
  *         description: Server status and welcome message
  */
 app.get("", (req: Request, res: Response) => {
-  return res.status(200).send({
-    message: "Welcome to Phantom server",
-    serverStatus: "RUNNING",
-  });
+    // req.setLocale("fr");
```

Comment:
Why these comments?

Reply:
I was testing with that however, it is now removed

Reply:
Yeah thank you